media_commons_booking_app/src/client/routes/booking/bookingProvider.tsx

```diff
@@ -72,6 +74,7 @@ export function BookingProvider({ children }) {
     console.log('isTrained from googlesheets', isTrained);
     setIsSafetyTrained(isTrained);
   }, [userEmail, safetyTrainedUsers]);
+  const isStudent = role === Role.STUDENT;
```

Comment:
this would recalculate every time any value in the provider changes, and there's a lot of values in here. That would mean any component using `isStudent` would also re-render! I'd put this line in a `useEffect` that updates a state value, or calculate this wherever we update the `role` state variable

Reply:
@lucia-gomez 
Oh you are right, I put it in useEffect.